ws-nextjs-app/pages/[service]/send/[id]/GenericMessage/index.tsx

```diff
@@ -10,14 +11,19 @@ interface Props {
 }
 
 const GenericMessage = ({ heading, children }: Props) => {
+  const el = useRef<HTMLHeadingElement>(null);
+  useEffect(() => {
+    el.current?.focus();
```

Comment:
I believe as part of the a11y AC the page should also scroll to the top when navigated to.

May be ok to do that here too? Just wouldn't want other folks to use this component and unexpectedly have the page automatically scroll to the top though. May be a non-issue since this component is within the Form folder, so something that could be changed if we ever move it out.

Reply:
Yep that's a good point. I have moved the useEffect/ useRef logic out to the Uploading Screen, Error Screen and Success Screen.